### PR TITLE
Handle invite event

### LINKED ISSUE #625: Invite messages don't show channel
"X invited Y into the channel" <-- you get this in the lobby, and it doesn't say what channel.

It would be good to either show the message in the channel itself, or to say

"X invited Y into the channel (#shout-irc)"

### PR DESCRIPTION
**Pull request by xPaw**
Fixes #625

### CODE REVIEW

File: client/js/shout.js [hunk 1]
```diff
@@ -205,6 +207,7 @@ $(function() {
 			"action",
 		].indexOf(type) !== -1) {
 			switch (type) {
+			case "invite": data.msg.formattedAction = "invited you to"; break;
```
Comment (YaManicKill) [inline client/js/shout.js]: So, is this just for invites that someone sends to you. What I was referring to was invites that are sent to someone else. Will this be both of them? If so, the wording doesn't work for all of them.

Comment (xPaw) [inline client/js/shout.js]: IRC protocol does not support showing when someone invites to the channel.

However there is http://ircv3.net/specs/extensions/invite-notify-3.2.html

**Comment by MaxLeiter** [comment]
For those curious, this is the FA icon used for invites: http://fortawesome.github.io/Font-Awesome/icon/envelope/

**Comment by erming** [comment]
Unfortunately I got this:

image

Running debug, the IRC message looks like this:
```
INVITE :erming!~erming@46.101.57.23 INVITE test-invites :#flerp
```

Maybe it works just as fine with `<name> invited you to #chan`?

**Comment by xPaw** [comment]
@erming should be fixed.

**Comment by astorije** [comment]
Hey @xPaw, would you mind copying this PR over at https://github.com/thelounge/lounge please?
Thanks!

**Pull request merged**